src/OrchardCore/OrchardCore.Data/OrchardCoreBuilderExtensions.cs

```diff
@@ -38,10 +38,10 @@ public static OrchardCoreBuilder AddDataAccess(this OrchardCoreBuilder builder)
                 services.AddScoped<IModularTenantEvents, AutomaticDataMigrations>();
 
                 // Adding supported databases
-                services.TryAddDataProvider(name: "Sql Server", value: "SqlConnection", hasConnectionString: true, hasTablePrefix: true, isDefault: false);
+                services.TryAddDataProvider(name: "Sql Server", value: "SqlConnection", hasConnectionString: true, sampleConnectionString: "The connection string to your database instance. e.g., Server=<em>localhost</em>;Database=<em>Orchard</em>;User Id=<em>username</em>;Password=<em>password</em>", hasTablePrefix: true, isDefault: false);
```

Comment:
Only the connection string should be in this object, and then in the view you can call `@T["The connection string to your database instance. e.g., ", provider.SampleConnectionString]`

Reply:
Got it 👍

Reply:
I really forgot the parameterized indexer :)